dash-renderer/src/components/error/FrontEnd/FrontEndErrorContainer.react.js

```diff
@@ -34,6 +34,12 @@ class FrontEndErrorContainer extends Component {
                         </strong>
                         ){connected ? null : '\u00a0 🚫 Server Unavailable'}
                     </div>
+                    <div
+                        className='dash-fe-error__icon-x'
+                        onClick={() => clickHandler()}
+                    >
+                        x
```

Comment:
```suggestion
                        ×
```
This is the `&times;` character, which makes a nicer button than the letter x.